hexrd/deprecation.py

```diff
@@ -21,6 +21,7 @@ def wrapper(*args, **kwargs):
                 print(
                     f"Warning: {func.__name__} is deprecated and is marked for"
                     f" removal. Please use {new_func} instead."
+                    f" Deprecation date: {deprecation_date}"
```

Comment:
Can you say `Removal date:` instead? I think "Deprecation date" sounds like the day it was deprecated.